Components/button/button.css

```diff
@@ -0,0 +1,100 @@
+.btn{
+    background-color: #FF3F6C;
+    color: white;
+    border: 0;
+    padding: 9px 15px;
+    font-size: 15px;
+    border-radius: 5px;
+    cursor: pointer;
+}
+
+.btn-outline{
+    background-color: white;
+    color: #FF3F6C;
+    border: 1px solid #FF3F6C;
+}
+
+.btn-outline:hover{
+    background-color: #FF3F6C;
+    color: white;
+}
+
+.btn-text{
+    background-color: transparent;
+    color: #FF3F6C;
+    text-decoration: underline;
+}
+
+.btn-icon{
+    color: #FF3F6C;
+    background-color: transparent;
+    border: 0;
+    font-size: 1.3rem;
+
+}
+
+.btn-primary{
+    background-color: #007BFF;
+}
+
+.btn-secondary{
+    background-color: #6C757D ;
+}
+
+.btn-success{
+    background-color: #28A745 ;
+}
+
+.btn-danger{
+    background-color: #DC3545 ;
+}
+
+.btn-warning{
+    background-color: #FFC107 ;
+}
+
+.btn-info{
+    background-color: #17A2B8 ;
+}
+
+.btn-dark{
+    background-color: #343A40 ;
+}
+
+.icon{
+    margin-right: 8px;
+    font-size: 1.2rem;
+}
+
+.btn-lg{
+    font-size: 18px;
+    padding: 1rem 1.1rem;
+}
+
+.btn-sm{
+    font-size: 13px;
+    padding: 8px 10px;
```

Comment:
Maintain consistency. Either use only rems or only px.

Reply:
I actually I was following as vikram said , using pixels for smaller pixel values and rem for more than 10px

Reply:
I will refactor it